include/mbedtls/config.h

```diff
@@ -2511,7 +2511,7 @@
  *            it, and considering stronger message digests instead.
  *
  */
-//#define MBEDTLS_MD2_C
+#define MBEDTLS_MD2_C
```

Comment:
You accidentally committed a modified `config.h`.

Reply:
Damn... sorry

Reply:
Fixed.